rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
+gem 'ffaker'
+gem 'sqlite3'
+gem 'stackprof'
+gem 'mysql2'
+gem 'pg'
+gem 'arel', github: 'rails/arel', branch: 'master'
```

Comment:
At Discourse we use the RAILS_MASTER hack 

https://github.com/discourse/discourse/blob/master/Gemfile